docs/development_suite/set-up-infrastructure/configure-provider.mdx

```diff
@@ -56,10 +61,14 @@ You can connect a provider by clicking on the "Add provider connection" button,
 ![Add provider 1a](img/add-provider-1a.png)
 ![Add provider 1b](img/add-provider-1b.png)
 
-In this step, you are required to choose the specific provider you want to use among the ones supported by the Console and, for the chosen provider, the capabilities you want to use it for ("Git Provider", "Secret Manager", "CI/CD Tool").
+In this step, you are required to choose the specific provider you want to use among the ones supported by the Console and, for the chosen provider, the capabilities you want to use it for ("Git Provider", "Secret Manager", "CI/CD Tool").  
+
+If you select a "Git Provider", you can also select its purpose of use, by choosing between "Project" (by default) or "Marketplace" functionalities.  
+If you preserve the default choice of "Project" functionality, the Git Provider will be specifically used for your Projects workspaces. By selecting "Marketplace", instead, you will be able to make any resources related to that Git Provider available to other Companies as well.  
```

Comment:
here we will insert a link to the documentation page in which we will describe how to connect a Marketplace resource to a Git Provider for Marketplace and make it available to other Companies.